src/components/modal/report/modal.jsx

```diff
@@ -124,120 +79,88 @@ class ReportModal extends React.Component {
-                                        type="button"
-                                        onClick={onRequestClose}
-                                    >
-                                        <div className="action-button-text">
-                                            <FormattedMessage id="general.close" />
-                                        </div>
-                                    </Button>
-                                ) : (
-                                    <Button
-                                        className={classNames(
-                                            'action-button',
-                                            'submit-button',
-                                            {disabled: !submitEnabled}
-                                        )}
-                                        {...submitDisabledParam}
-                                        key="submitButton"
-                                        type="submit"
-                                    >
-                                        {isWaiting ? (
-                                            <div className="action-button-text">
-                                                <Spinner />
-                                                <FormattedMessage id="report.sending" />
-                                            </div>
-                                        ) : (
-                                            <div className="action-button-text">
-                                                <FormattedMessage id="report.send" />
-                                            </div>
-                                        )}
-                                    </Button>
-                                )}
-                            </div>
-                        </FlexRow>
-                    </Form>
+                                <TextArea
+                                    autoFocus
+                                    required
+                                    className="report-text"
+                                    elementWrapperClassName="report-modal-field"
+                                    label={null}
+                                    name="notes"
+                                    validationErrors={{
+                                        isDefaultRequiredValue: intl.formatMessage({id: 'report.textMissing'}),
+                                        maxLength: intl.formatMessage({id: 'report.tooLongError'}),
+                                        minLength: intl.formatMessage({id: 'report.tooShortError'})
+                                    }}
+                                    validations={{
+                                        maxLength: 500,
+                                        minLength: 20
+                                    }}
+                                />
+                            </FormStep>
+                            <FormStep
```

Comment:
Could you put a comment before this line, making clear that it constitutes the content of the `confirmation` step?

Reply:
Done